src/sections/Company/News-grid/index.js

```diff
@@ -1,19 +1,104 @@
-import React from "react";
+import React, { useState } from "react";
 import { Container, Row, Col } from "../../../reusecore/Layout";
 import Card from "../../../components/Card";
 import PageHeader from "../../../reusecore/PageHeader";
 import { NewsPageWrapper } from "./NewsGrid.style";
 import rss_feed_icon from "../../../assets/images/socialIcons/rss-sign.svg";
+import Button from "../../../reusecore/Button";
+import { useEffect } from "react";
 
+var CoverageFiltered= true;
+var PressReleaseFiltered=true;
+
+function colorchange(id) {
```

Comment:
Could make this a reusable function and import it wherever required to make this more robust? cc @Nikhil-Ladha your views?

Reply:
I can make this reusable by implementing it in the custom button component that we made.

Reply:
camelCase